esp/platform/txsummary.cpp

```diff
@@ -21,10 +21,16 @@
 #include <algorithm>
 
 using std::find_if;
+using std::for_each;
 
 #define VALIDATE_KEY(k) if (!(k) || !(*k)) return false
 #define MATCH_KEY       [&](const Entry& entry) { return stricmp(entry.key.str(), key) == 0; }
 
+bool operator < (const StringBuffer& a, const StringBuffer& b)
```

Comment:
why do we need to provide this operator definition? and if it's needed, do we also need < >=, etc?